src/store/slices/MoviesSlice.ts

```diff
@@ -6,14 +6,12 @@ import { IMovie } from "../../types/movie";
 interface IState {
```

Comment:
the file should start with the small letter

Reply:
done